example/lib/home.dart

```diff
@@ -24,7 +24,8 @@ class AnylineDemoApp extends StatelessWidget {
       theme: ThemeData.light().copyWith(
```

Comment:
The changes in this file are helpfully reapplied each time by Visual Studio (auto-format) whenever I visit it.